src/test/java/com/oath/halodb/EstimatedHistogramTests.java

```diff
@@ -0,0 +1,121 @@
+package com.oath.halodb;
+
+import com.oath.halodb.histo.EstimatedHistogram;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.testng.internal.junit.ArrayAsserts.assertArrayEquals;
+
+public class EstimatedHistogramTests {
+
+    @Test
+    public void testGetBuckets() {
+        EstimatedHistogram estimatedHistogram = new EstimatedHistogram(3);
```

Comment:
Is it possible to initialize estimatedHistogram with some values, so it would be possible to show the difference between `true` and `false`, namely that `true` resets the contents?

Reply:
Done